docs/howto/connection/oracle.rst

```diff
@@ -85,3 +85,36 @@ Extra (optional)
     .. code-block:: bash
 
         export AIRFLOW_CONN_ORACLE_DEFAULT='oracle://oracle_user:XXXXXXXXXXXX@1.1.1.1:1521?encoding=UTF-8&nencoding=UTF-8&threaded=False&events=False&mode=sysdba&purity=new'
+
+
+Tips and Tricks for Oracle Shops
+================================
+
+Oracle users an SID to identify a unique database instance. Oracle also has an unusual definition of schema and user, wherein a schema set of objects owned by a user. Therefore, creating a connection through the Airflow web GUI can be confusing.
+
+To create a connection: 
+
+- Go to Admin => Connections => Create
+
+- Name your connection
+
+- Select connection type: Oracle
+
+- Enter your hostname in the following format: NameOrIP:PORT#/SID Example(s): 10.0.0.111:1521/orcl or weshouldhaveusedpostgres.ourdomain.com:1521/dev
+
+- Enter your username in both the schema and login fields
+
+- Enter your password in the password field.
+
+- Save and continue.
+
+If RBAC is not enabled do the following:
```

Comment:
RBAC is the only supported UI on the master branch.

Reply:
Removed that part.